Update documentation (#4)
FindHotel/fh-app-assignment: INSTRUCTIONS.md

Check the following screenshots for the designs to build. You can focus on the Guest and Room Selector, **building the home screen is not required**.

| Home screen                     | Guest and Room Overlay                                                      | Guest and Room Overlay with more rooms                                                                         |
| ------------------------------- | --------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------- |
| ![Home](Home.jpg "Home screen") | ![Guest and Room Overlay](GuestPicker-Default.jpg "Guest and Room Overlay") | ![Guest and Room Overlay with more rooms](GuestPicker-More-Rooms.jpg "Guest and Room Overlay with more rooms") |

### Time to spend on the assignment ⏳

We don't assign a fixed time deadline for it, but after you’ve read the assignment, please let us know when you’re planning to complete it. After you complete the assignment we'll review it internally.

> - if we evaluate it to be positive, we'll schedule a technical interview to discuss the results together
> - if we evaluate otherwise, we'll share our feedback and findings with you

Reviewer: I couldn't suggest the complete line, but I would separate the time statement from the evaluation, and put this after the "concessions" statement in the next line.

```suggestion
### Evaluation and next steps

- if we evaluate your assignment positively, we'll schedule a technical interview to discuss the results together,
- otherwise, we'll share our feedback and findings with you
```

##### Examples

> - `1:4,6|3` → Two rooms: one room with one adult and two children ages four and six; and the other room with three adults and no children

Reviewer: ```suggestion
- `1:4,6|3` → Two rooms: one room with one adult and two children aged four and six; and the other room with three adults and no children
```

- [ ] **Configurability:** the functional requirements can change in the future, do what you can to make sure the component is easily configurable.
- [ ] **Usability:** should be evident to the user how to use the guest and room selector. We are here to help the user to understand what are their options and guide them.
- [ ] **Testability:** we recommend to write automated tests (unit or integration) to verify and document the business logic. We leave to you to decide and [document](DOCUMENTATION.md) which parts of the implementation should be covered by tests, given the constraint of time.
- [ ] **State management and scalability:** use any state management mechanism you’d like, keeping in mind how you would scale the architecture of the application as it grows.
- [ ] **Deep-linking and navigation:** even though the home view doesn't need to be implemented in detail, please showcase how you would handle navigation and deep-linking in your application.

### Technology and Frameworks

The list below is based on what we are currently using at FindHotel. If you have other technology or frameworks in mind, you’re free to use it.

> We kindly ask you to [document](DOCUMENTATION.md) your decisions and guide us through how to easily debug and run the application in iOS and Android simulators.

Reviewer: <3

- Exercise the component interactively with an appropriate test library
- Use git best practices, commit messages and such

### Not required

> - Server side or API: for this assignment you are not required to build any server side or API logic, you can build all logic in the application.

Reviewer: ```suggestion
- Server side or API: for this assignment you are not required to build any server side or API logic, you can build all logic in the client application.
```
